Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Alamofire
+
+/// Refunds: Remote Endpoints
+///
+public class RefundsRemote: Remote {
```

Comment:
Would it make sense to make this class final? It seems like we are not going to subclass it anyway, and at a given moment in time there seemed to be performance advantages in doing so: https://developer.apple.com/swift/blog/?id=27